pkg/porter/explain.go

```diff
@@ -464,3 +465,38 @@ func (p *Porter) printDependenciesExplainTable(bun *PrintableBundle) error {
 		}
 	return printer.PrintTable(p.Out, bun.Dependencies, printDependencyRow, "Alias", "Reference")
 }
+
+func (p *Porter) printInstallationInstructionBlock(bun *PrintableBundle, bundleReference string) error {
+	fmt.Fprint(p.Out, "To install this bundle run the following command, passing --param KEY=VALUE for any parameters you want to customize:\n")
+
+	var bundleReferenceFlag string
+	if bundleReference != "" {
+		bundleReferenceFlag += " --reference " + bundleReference
+	}
+
+	// Generate any predefined credentials first
+	for _, credential := range bun.Credentials {
+		fmt.Fprintf(p.Out, "porter credentials generate %s%s\n", credential.Name, bundleReferenceFlag)
+	}
+
+	// Bundle installation instruction
+	var requiredParameterFlags string
+	for _, parameter := range bun.Parameters {
+		if parameter.Required {
+			requiredParameterFlags += parameter.Name + "=TODO "
+		}
+	}
+
+	if requiredParameterFlags != "" {
+		requiredParameterFlags += " --param "
+	}
+
+	var credentialFlags string
```

Comment:
This can be a conditional "if there are creds then include a single -c mycreds flag", otherwise don't print anything.